htdocs/core/modules/modFournisseur.class.php

```diff
@@ -253,6 +254,30 @@ public function __construct($db)
 		$this->rights[$r][4] = 'commande';
 		$this->rights[$r][5] = 'export';
 
+		$r++;
+		$this->rights[$r][0] = 1192;
+		$this->rights[$r][1] = 'Self-Approve LVL1';
+		$this->rights[$r][2] = 'r';
+		$this->rights[$r][3] = 0;
+		$this->rights[$r][4] = 'commande';
+		$this->rights[$r][5] = 'selfapprovelvl1';
```

Comment:
As this need an more complete setup in permission that standard rule (can read / update /delete), the new permissions must have key  
$this->rights[$r][5] = 'selfapprovelvl1_advance';
with _advance at end of key, so the permission will be visible when the option "Use advanced permission" is on.